src/css/utils/display.css

```diff
@@ -0,0 +1,52 @@
+.none {
+  display: none !important;
+}
+
+.js-none {
+  @media (--js) {
+    display: none !important;
+  }
+
+  .js & {
+    display: none !important;
+  }
```

Comment:
Check if there's a PostCSS plugin doing the class addition, the same way there's one for light/dark mode.

Reply:
Properly.